backend/consensus/base.py

```diff
@@ -1911,6 +1913,10 @@ async def handle(self, context):
                 context.transaction.hash, None
             )
             context.transaction.timestamp_appeal = None
+            context.transactions_processor.set_transaction_appeal_failed(
```

Comment:
@kstroobants doesn't `context.transaction.appeal_undetermined` mean that the appeal failed?